src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java

```diff
@@ -221,7 +221,8 @@ private void validateProjectState() {
         if (props.getDescriptionsExpandedDate() == null) {
             throw new InvalidProjectStateException("Descriptions must be created and expanded");
         }
-        if (props.getSourceFilesUpdatedDate() == null) {
+        if (props.getSourceFilesUpdatedDate() == null || (props.getSourceFilesUpdatedDate() == null
+                && !streamingMetadataService.hasProjectStreamingMetadataField())) {
```

Comment:
I'm actually not sure we need this exception. Users can generate a blank source file mapping if need be, and it's a little hard to saw what the repercussions further down the pipeline would be for not having a source files mapping at all.